webroot/plots/plots.org

```diff
@@ -1198,11 +1410,37 @@ saveWidget(p, selfcontained = FALSE, file.path(getwd(), "plotly/", "io500-mdtest
 #+end_src
 
 #+RESULTS:
-: 
-: Warning message:
-: 'scattergl' objects don't have these attributes: 'hoveron'
-: Valid attributes include:
-: 'type', 'visible', 'showlegend', 'legendgroup', 'name', 'uid', 'ids', 'customdata', 'meta', 'selectedpoints', 'hoverinfo', 'hoverlabel', 'stream', 'transforms', 'uirevision', 'x', 'x0', 'dx', 'y', 'y0', 'dy', 'text', 'hovertext', 'textposition', 'textfont', 'mode', 'line', 'marker', 'connectgaps', 'fill', 'fillcolor', 'selected', 'unselected', 'opacity', 'hovertemplate', 'texttemplate', 'error_x', 'error_y', 'xcalendar', 'ycalendar', 'xaxis', 'yaxis', 'idssrc', 'customdatasrc', 'metasrc', 'hoverinfosrc', 'xsrc', 'ysrc', 'textsrc', 'hovertextsrc', 'textpositionsrc', 'hovertemplatesrc', 'texttemplatesrc', 'key', 'set', 'frame', 'transforms', '_isNestedKey', '_isSimpleKey', '_isGraticule', '_bbox'
+#+begin_example
+
+Warning messages:
+1: 'scattergl' objects don't have these attributes: 'hoveron'
```

Comment:
It isn't clear to me (because I really do not understand this code, no fault of yours) whether all of these warning messages are a sign of something broken in the tool that is generating these tables, or if this is expected behavior?  It would really be useful for my (and presumably other's) understanding of what is going on with this commit if there were a few sentences of description in the commit message to explain the intent of this change and anything noteworthy going on.  While that isn't strictly necessary for "boring" changes such as my recent PRs that are just changing the text of a few pages, some minimal explanation of the guts of this code would be very welcome.

Reply:
This is expected, as the HTML files with the interactive plots (using Plotly) are automatically generated in R.